app/models/speaker.py

```diff
@@ -87,6 +88,7 @@ def __init__(self,
         # ensure links are in social fields
         self.ensure_social_links()
         self.user = user
```

Comment:
Try remove it. Your issue will be solved. Since this is a relationship reference I don't it is required.